src/libsync/theme.cpp

```diff
@@ -106,6 +106,16 @@ QString Theme::appName() const
     return APPLICATION_SHORTNAME;
 }
 
+QString Theme::currentAccountStateIndicatorOnlineImageSource() const
```

Comment:
What about changing those to return QUrl? And then adding the qrc scheme in here?

This way this would avoid the duplication on the QML side and move the responsibility of always having well formed URLs back with the C++.